### DISCUSSION TITLE
Categories limited to 5 pages

### DISCUSSION THREAD

**Discussion by Strongeyed** [swellstores/community#180] [Help] [answered]
I've had this issue launched with the helpdesk over 6 months ago but this might not be something many people have issues with as I couldn't find another ticket in here.
 
When you have more than 120 products in one category to display to customers the available pages to browse stop at 'Page 5 of 5'. 

This limits the amount of products you can display to customers per category.

As example; each categories page shows 24 products, having a maximum of 5 pages this means you can display a total of 120 products per category to customers. However I have categories with over 200 products, which I'd require customers to browse more than 5 pages or display more products per page so I can make up for the 5 page limit.

Example of the 5 page limit, this category has over 200 unique products.
<img width="1276" alt="Screenshot 2023-03-18 at 9 48 31 PM" src="https://user-images.githubusercontent.com/128098520/226095421-5ddc53eb-529c-4bc1-8281-30e2eeca73e4.png">

**Comment by rodrigoddalmeida** [comment, marked as answer]
@Robin-DG a fix should be released just now @ v2.0.15

**Reply by Strongeyed** [reply]
Thank you @rodrigoddalmeida it works and shows in my DEV site, however now I need the other bug to be fixed to be able to actually publish the changes as my storefront is not publishing. I've seen that quite a couple of people have that issue at the moment, is there a way to fix this?

**Reply by rodrigoddalmeida** [reply]
@Robin-DG could you try publishing again - we have just deployed fixes to the publishing action.

**Reply by Strongeyed** [reply]
Hi @rodrigoddalmeida no luck unfortunately, still doing the same thing where it jumps from 0% to changes are live, normally my publishing process would take about 15 minutes. Would you like me to message you my store link?
